src/main/java/com/iexec/sms/tee/config/TeeWorkerPipelineConfiguration.java

```diff
@@ -45,11 +45,12 @@ public record Pipeline(
             @NotNull(message = "Post-compute configuration must not be null") @Valid StageConfig postCompute
     ) {
         public TeeServicesProperties toTeeServicesProperties(final String lasImage) {
+            final String teeFrameworkVersion = version();
             final TeeAppProperties preComputeProperties = preCompute().toTeeAppProperties();
             final TeeAppProperties postComputeProperties = postCompute().toTeeAppProperties();
```

Comment:
Here, we directly use fields from the record and do not need to do it through the getter method.
As there is no transformation, the `teeFrameworkVersion` local variable is unnecessary as well and can be replaced with `version.`
```suggestion
            final TeeAppProperties preComputeProperties = preCompute.toTeeAppProperties();
            final TeeAppProperties postComputeProperties = postCompute.toTeeAppProperties();
```

Reply:
Resolved in [75e68d2](https://github.com/iExecBlockchainComputing/iexec-sms/pull/289/commits/75e68d2b2c9b33c73e83b23f2d933a4758ff25e0)

Reply:
I still see this in code
```
            final TeeAppProperties preComputeProperties = preCompute().toTeeAppProperties();
            final TeeAppProperties postComputeProperties = postCompute().toTeeAppProperties();
```

Reply:
Resolved in [54c5ec3](https://github.com/iExecBlockchainComputing/iexec-sms/pull/289/commits/54c5ec34f87655f3ed3219e593c4847c4c32d3a3)